src/Features/LanguageServer/Protocol/Features/CodeFixes/CodeFixService.cs

```diff
@@ -136,13 +140,27 @@ internal partial class CodeFixService : ICodeFixService
                 }
             }
 
+            var copilotDiagnostics = await GetCopilotDiagnosticsAsync(document, range, priorityProvider.Priority, cancellationToken).ConfigureAwait(false);
+            var convertedCopilotDiagnostics = ConvertToMap(text, copilotDiagnostics);
+            var spanToCopilotDiagnostics = new SortedDictionary<TextSpan, ImmutableArray<DiagnosticData>>();
```

Comment:
why sorted?

Reply:
Following the same pattern as other diagnostic buckets in this method.